### ISSUE TITLE
LICENSE?

### ISSUE THREAD

**Issue by selfup** [nanosai/stream-ops-java#5]
There doesn't seem to be a LICENSE to indicate legal ramifications for either the consumer or the producer. Is this just Open Core?

**Comment by jjenkov** [comment]
You are right! ... although I was certain that I chose "Apache License 2.0" when I created the repository originally. I will address this pretty soon ! Thank you for letting us know!
